### ISSUE TITLE
Relation between People, Customers, Organizations

### ISSUE THREAD

**Issue by hamid80386** [venturedrake/laravel-crm#59]
- Laravel CRM Version: #.#.#
- PHP Version:

### Description:
I just couldn't find relation between people, organization and customer.
In my experience in a CRM software we have a customer that has relation to organization, and many customer can be related to  a organization.
but you don't do that, you make them separately, and after that, when you are in creating new lead, you get Customer name again, Organization Again and people again. whereas you could make a relation between people and organization and remove customer part, and then in making new lead, call just a people and it load it's organization automatically.

Because lack of enough documents in doc part, this question happened to me.

Please tell us about Customer, Organization and people and their rules and relations in your system.

thanks

### Steps To Reproduce:

**Comment by andrewdrake** [comment]
Sorry I haven't quite finished the docs as yet, working on it.

A customer is more of a group of people or organizations. So a customer has one or many people or organizations. 

A customer for example could be a person, but that person has 3 organizations (companies). The customer or client is that person you deal with, but you may raise an invoice against that customer's organization.

Open to suggestions here, I think it may be a bit confusing.

**Comment by hamid80386** [comment]
I agree with this perspective, as it's indeed possible for an individual to work for multiple companies, be responsible for procurement for multiple companies, or even work independently.

The only caveat is that this needs to be clearly communicated to users.

Thank you for the clarification.

**Comment by hamid80386** [comment]
I went back to the customer section
Do you think the fields in this section are not enough?
Doesn't he also want relations and address and phone and tag
The customer is in this B2B system, but it is mandatory to add a person and organization in the leads section and other sections
I think we have violations or complications in some places, a few short explanations in the documentation section can be very helpful

**Comment by andrewdrake** [comment]
I am re-evaluating this design at the moment

**Comment by juanmafx** [comment]
Hi hope you are doing well.I would like to know withch it's the default user admin password. At que moment of start up environment.Kind regards.On May 22, 2024 03:50, Andrew Drake ***@***.***> wrote:
I am re-evaluating this design at the moment

**Comment by andrewdrake** [comment]
There is no default admin user, you can register users as you normally would in your Laravel app at /register

There is a config setting also where you can set an initial admin user to access the crm, this is the section from the laravel-crm.php config file. 

` /*
    |--------------------------------------------------------------------------
    | CRM Owner
    |--------------------------------------------------------------------------
    |
    | This value relates to the primary owner for the crm. It must be set as
    | the email address for the user registered in the users table so that you
    | can access the crm initially. You will need to register this user after
    | the crm is installed if not already.
    |
    */

    'crm_owner' => env('LARAVEL_CRM_OWNER', ''),`

**Comment by andrewdrake** [comment]
Docs are now complete
